src/redux/actions/userActions.js

```diff
@@ -0,0 +1,11 @@
+import * as userApi from "api/usersApi";
+
+export function signUp(user) {
+  return async (dispatch, getState) => {
```

Comment:
```suggestion
  return async () => {
```

You are not using these functions